Support filter argument for `nerdctl images` command (#1307)
containerd/nerdctl: README.md

  - :nerd_face: `--format=wide`: Wide table
  - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
- :whale: `--digests`: Show digests (compatible with Docker, unlike ID)
- :whale: `-f, --filter`: Filter the images
  - :whale: `--filter=before=<image:tag>`: Images created before given image (exclusive)

>   - :whale: `--filter=since=<image:tag>`: Images created after given image (exclusive)

Reviewer: Could you document unimplemented filters such as “label”?

Author: Done

  - :nerd_face: `--format=wide`: Wide table
  - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
- :whale: `--digests`: Show digests (compatible with Docker, unlike ID)
- :whale: `-f, --filter`: Filter the images. For now, only 'before=<image:tag>' and 'since=<image:tag>' is supported.
  - :whale: `--filter=before=<image:tag>`: Images created before given image (exclusive)
  - :whale: `--filter=since=<image:tag>`: Images created after given image (exclusive)
- :nerd_face: `--names`: Show image names

> For now, the following argument for `--filter` is not supported yet:

Reviewer 2: ```suggestion
Following argument for `--filter` are not supported yet:
```

  - :nerd_face: `--format=wide`: Wide table
  - :nerd_face: `--format=json`: Alias of `--format='{{json .}}'`
- :whale: `--digests`: Show digests (compatible with Docker, unlike ID)
- :whale: `-f, --filter`: Filter the images. For now, only 'before=<image:tag>' and 'since=<image:tag>' is supported.
  - :whale: `--filter=before=<image:tag>`: Images created before given image (exclusive)
  - :whale: `--filter=since=<image:tag>`: Images created after given image (exclusive)
- :nerd_face: `--names`: Show image names

Following arguments for `--filter` are not supported yet:

1. `--filter=label=<key>=<value>`: Filter images by label
2. `--filter=reference=<image:tag>`: Filter images by reference
3. `--filter=dangling=true`: Filter images by dangling

> Unimplemented `docker images` flags: `--filter`

Reviewer: You can now drop this line
